src/bindings/python/config.py

```diff
@@ -0,0 +1,39 @@
+from pathlib import Path
+from typing import Union
+
+from .valhalla_build_config import config as default_config, Optional
+
+def _sanitize_config(dict_: dict = None) -> dict:
+    """remove the "Optional" values we get back from the config generator"""
+
+    int_dict_ = dict_.copy()
+    for k, v in int_dict_.items():
+        if isinstance(v, Optional):
+            del dict_[k]
```

Comment:
we should add a method to the config generation that does this like `--minimal` or something